app/src/test/java/com/x8bit/bitwarden/data/vault/repository/VaultRepositoryTest.kt

```diff
@@ -4365,7 +4369,42 @@ class VaultRepositoryTest {
         } just runs
 
         val syncResult = vaultRepository.syncForResult()
-        assertEquals(SyncVaultDataResult.Success, syncResult)
+        assertEquals(SyncVaultDataResult.Success(itemsAvailable = true), syncResult)
+    }
+
+    @Suppress("MaxLineLength")
+    @Test
+    fun `syncForResult should return success result with itemsAvailable = false when sync succeeds and ciphers list is empty`() = runTest {
```

Comment:
Wanna run the formatter here, even with the suppression, `runTest {` should be on it's own line